%% Template for a preprint Letter or Article for submission
%% to the journal Nature.
%% Written by Peter Czoschke, 26 February 2004
%%

\documentclass{nature}


%% make sure you have the nature.cls and naturemag.bst files where
%% LaTeX can find them

\bibliographystyle{naturemag}

\title{Biogenic methane from phosphonates tracks parent carbon source.}

%% Notice placement of commas and superscripts and use of &
%% in the author list

\author{Lina Taenzer$^{1,2}$, Paul Carini$^2$, Sara Lincoln$^3$, Janel Gaube$^1$, \& William D. Leavitt$^{1,4,5,*}$}

\usepackage{amsmath}

\begin{document}

\maketitle

\begin{affiliations}
 \item Department of Earth Sciences, Dartmouth College Hanover, NH, USA 03755 
 \item Department of Biology(?), University of Arizona, 
 \item Department of Earth Science, Penn State…
 \item Department of Biological Sciences, Dartmouth College Hanover, NH, USA 03755
 \item Department of Chemistry, Dartmouth Colleg Hanover, NH, USA 03755
\end{affiliations}

\begin{abstract}
Methane is a potent and pervasive greenhouse gas produced by an array of biogenic and abiogenic sources. Methane has been found supersaturated in the fully oxygenatedthe surface of oceans and lakes, supersaturated with respect to the overlying atmosphere.  Multiple biogenic pathways have been proposed as the possible sources for supersaturated methane found in oxygen rich watersof this mysterious methane. Among these possible sources is the most likely identified is the bacterial demethylation of alkylphosphonates via the enzyme C-P lyase, releasing the greenhouse gas as a byproduct of phosphorous acquisition by phosphate-starved heterotrophs and even some primary producers.  Recent work with some of the oceans most abundant bacterial photoautotrophs and heterotrophs has demonstrated that methane is released as a by-product C-P lyase containing microbes with dissolved organic matterof this reaction; however, tracing the contribution of this methane to ocean, lake and ultimately atmospheric pools first requires a quantitative and unique metric of this methane, but the isotopic fractionation imparted to the product methane has remained a mystery. The stable isotopic composition and fractionation from the source material represents a quantitative means to identify this methane.  Considering that the $\delta$ $^13$C  composition of CH4 is routinely used to infer the process of methane formation, Tthe lack of information regarding phosphonate-derived methane generated methane is a critical gap in our understanding biological sources of methane sources.  To address this we cultured an array of cosmopolitan heterotrophic marine and freshwater bacterial strains conducting containing the this metabolism and analyzedCP-lyase enzyme and determined the carbon isotopic fractionation between methane for its $\delta$ $^13$C as a means to track this processand substrate. Our results show that $\delta$ $^13$C of C-P lyase produced CH4 mirrors that of substrate (fractionation= XX). A compilation of water-column methane isotope data suggest that thisThis pathway could is the most probable thus be the source of isotopically light methane found coinciding with methanein oxygenated surface oceans. By examining  ocean and lake metagenome data, it is further evident that  maxima. In light of this thewe argue that bacterial degradation of MPn may beis a probable widespread and significant source of methane in oxygenated marine waters, as well asand may also play a critical role in lakes and soils where inorganic phosphorous is limiting, and/or alkylphosphonates are abundant. This methane from such sources may contribute a critical flux from aerobic waters to the atmospheric methane reservoir with a distinctive isotopic signal.  
\end{abstract}

Surface waters supersaturated in methane is a yet to be explained phenomenon commonly referred to as the \textit{methane paradox}.  Observations of maximum dissolved methane concentrations in surface oxygen-rich water layers is mysterious as it starkly juxtaposes the anoxic requirements of canonical methane production pathways. The conventional understanding of dissolved methane production in oceans and lakes requires microbiological activity and or abiogenic abiotic water-rock interactions.Since canonical microbiological methane sources involve enzymes that operate under require strict anoxia to operate (e.g. organic rich sediments or upwelling zones) and geogenic sources are relegated to ophiolitic systems the source of this methane is a mystery. Previously suggested explanations include production in micro-anoxic zones (e.g. fecal pellets, animal guts, etc..), and input from surrounding anoxic water or sediments. Considering that dissolved methane and oxygen reach coinciding maximum concentrations in surface waters, the methane in these areas can only be explained by an active source of production - which micro-anoxic zones are unlikely to quantitatively account for.  More recently, research groups have proposed a production mechanism by oxygen tolerant methanogens that directly inhabit phosphorous (P)- depleted surface waters. While past studies have characterized the $\delta$ $^13$C of canonical methane producing processes as a means to track relevant production sources in the environment, given the novelty of the C-P lyase hypothesis as an explanation to the methane paradox a characterization of this metabolism is still lacking. Here we analyze the $\delta$ $^13$C of methane produced by C-P lyase metabolism to determine whether it could serve as the enigmatic source in the long-standing \textit{methane paradox}. 

More recently, research groups have proposed a production mechanism by oxygen tolerant methanogens that directly inhabit phosphorous (P)- depleted surface waters. Phosphorous is an essential nutrient for life, and is a growth-limiting factor in large areas of the world's surface oceans and certain lacustrine systems. When organic surface water phosphate concentrations are too low to support growth, bacteria encoding the C-P lyase enzyme complex are believed to de-methylate biologically produced alkylated phosphonate compounds as an alternative P-source (REF: Karl et al. 2008). The source of alkylphosphonates, such as MPn, in surface waters are pools of high-molecular weight dissolved organic matter (HWDOM) (c.f. Repeta et al. 2016). The C-P lyase is a well-known enzyme complex that is broadly utilized for the degradation of alkylphosphonates. A by-product of this metabolism is methane (Eq. 1). 
Equation 1. CH3POH2 + H2O  CH3 + H3PO4
This reaction is a compelling explanation for CH4  found in oxygen-rich waters for several reasons: (1) the C-P lyase enzyme complex is uninhibited by oxygen, and encoded for by a range of marine and freshwater bacteria (Fig 1.), (2) evolution of CH4 has been tracked in experiments where bacteria were cultured on MPn-containing media, and (3) MPn-like alkyphosphonates are found in naturally occurring HMWDOM and the genes for its synthesis have been identified in one of the most abundant marine archeon, Nitrosopumulis maritimus (REF: Metcalf et al. 2012). 

Past studies have characterized the $\delta$ $^13$C of canonical methane producing processes as a means to track production sources present in any given environment. Characteristic values of surface-layer dissolved methane tend to be isotopically lighter in $\delta$ $^13$C than methane produced biologically from acetate fermentation and carbon dioxide reduction. Given the novelty of the C-P lyase hypothesis as an explanation to the methane paradox, data characterizing the $\delta$ $^13$C of methane produced by this metabolism is lacking.  Using batch culture experiments to culture various strains, we captured methane across experimental growth and analyzed $\delta$ $^13$C of methane produced by C-P lyase metabolism. Through a comparison of global $\delta$ $^13$C values of methane in oxygenated waters we access the importance of this source to oxic water layers. 

Figure 1 The C-P lyase gene cluster is found worldwide.

\textbf{Results} We cultured four bacterial strains (Table 1.) isolated from a lake, and the common marine strain Pseudomonas stutzeri on a nutrient MOPS media that contained MPn as the sole source of phosphorous. Upon establishing initial growth, we tracked the methane produced along the growth trajectory of each strain of bacteria. To analyze the fractionation in $\delta$ $^13$C between MPn and methane we subsampled gas from the headspace of growth bottles, capturing lag, exponential, and stationary growth phases. 
We observed no significant differences in the $\delta$ $^13$C signature of methane produced by different strains. Methane accumulated in the headspace of experimental bottles increased corresponding with growth of bacteria but did not change in carbon isotopic composition. To expand the scope of investigation of this metabolism we grew Pseudomonas stutzeri on nitrate (instead of oxygen), at three different temperatures: 10˚C, 30˚C, and 37˚C. Shifting the temperature away from optimal environmental conditions stressed the organisms (evidenced by a notable decrease in growth rate of P. stutzeri at 10˚C (Fig. 1)), but did not coincide with a change in  $\delta$ $^13$C fractionation. Interestingly, regardless of strain, oxygen/nitrate based growth, or temperature, C-P lyase metabolism produced methane with a $\delta$ $^13$C value that effectively mirrored that of the MPn used in the media. 

PWe exhaustively analyzed microbial methane production from MPn, seeking potential causes for varying signals in fractionation. Our analysis of methane produced by various bacterial strains, and at differing growth temperatures, indicates that methane produced as a result of the de-methylation of MPn under organic P-starved simulated natural environmental conditions quantitatively tracks bacterial growth and with a $\delta$ $^13$C consistent with that of the substrate. This provides analytical support for the mechanism of the C-P lyase reaction proposed by Kamat et al (2013), where the complete methyl group of the precursor MPn molecule is transferred to product methane. 
\textbf{Discussion.} Our results indicate that the C-P lyase mechanism produces methane that can be distinguished based on its minimal fractionation in $\delta$ $^13$C (VALUE) in comparison with other biological production sources in the environment such as hydrogenotrophic (VALUE) and aceticlastic (VALUE) methanogenesis (Fig XXX.). 
Average surface-layer dissolved methane (VALUE) is characteristically lighter in $\delta$ $^13$C than methane produced under natural environmental conditions by acetate fermentation or carbon dioxide reduction. 
 
Here we have determined that aerobic bacteria that de-methylate methylphosphonic acid via the C-P lyase produce methane that is not fractionated in $\delta$ $^13$C composition to that of the substrate. This means that in natural systems methane produced by respective aerobic bacteria should be nearly identical of the $\delta$ $^13$C of the MPn in the HWDOM pool in surface waters.

Since DOC values in surface waters have a (XXX) the biodegradation of MPn, on a chemical basis, is uniquely able to account for the needed active aerobic source in oxygenated waters as well as reconcile the previously enigmatic lighter $\delta$ $^13$C signature of surface water dissolved methane.
\textbf{Conclusion.} The microbial formation of methane from alkyl phosphonates the fully oxygenated surface waters of oceans and lakes is now traceable by the isotopic signature that we have determined herein. Beyond the relevance of this mechanism in surface oceans or stratified lacustrine environments, this production pathway could furthermore serve as an explanation for large accumulations of methane recently discovered in wetlands. A combination of environmental metagenomic sequencing for C-P lyase, as well as carbon isotopic analysis of organic matter and dissolved methane , and will aid in estimating the global contribution of this methane source, providing new insight and understanding into the global methane cycle.  


\begin{figure}
%%%\includegraphics{something} % this command will be ignored
\caption{Each figure legend should begin with a brief title for
the whole figure and continue with a short description of each
panel and the symbols used. For contributions with methods
sections, legends should not contain any details of methods, or
exceed 100 words (fewer than 500 words in total for the whole
paper). In contributions without methods sections, legends should
be fewer than 300 words (800 words or fewer in total for the whole
paper).}
\end{figure}

\begin{methods}


\end{methods}

%% Put the bibliography here, most people will use BiBTeX in
%% which case the environment below should be replaced with
%% the \bibliography{} command.

% \begin{thebibliography}{1}
% \bibitem{dummy} Articles are restricted to 50 references, Letters
% to 30.
% \bibitem{dummyb} No compound references -- only one source per
% reference.
% \end{thebibliography}

\bibliographystyle{naturemag}
\bibliography{sample}


%% Here is the endmatter stuff: Supplementary Info, etc.
%% Use \item's to separate, default label is "Acknowledgements"

\begin{addendum}
 \item Put acknowledgements here.
 \item[Competing Interests] The authors declare that they have no
competing financial interests.
 \item[Correspondence] 
\end{addendum}

%%
%% TABLES
%%
%% If there are any tables, put them here.
%%

\begin{table}
\centering
\caption{This is a table with scientific results.}
\medskip
\begin{tabular}{ccccc}
\hline
1 & 2 & 3 & 4 & 5\\
\hline
aaa & bbb & ccc & ddd & eee\\
aaaa & bbbb & cccc & dddd & eeee\\
aaaaa & bbbbb & ccccc & ddddd & eeeee\\
aaaaaa & bbbbbb & cccccc & dddddd & eeeeee\\
1.000 & 2.000 & 3.000 & 4.000 & 5.000\\
\hline
\end{tabular}
\end{table}

\end{document}
